docs/configuration.md

```diff
@@ -144,6 +144,11 @@ Based on the delivery method type, additional parameters are required:
     - `sender_email` - the sender email for notifications and digests. Must be verified in AWS SES.
 * AWS_SNS:
     - No additional parameters needed. Target SNS topic Arn is configured in recipients section.
+* SMTP:
+    - `sender_email` - the sender email for notifications and digests.
+    - `credentials_secret_name` - the name of the secret stored in AWS Secrets Manager containing the SMTP server credentials. Required key-value pairs: `SMTP_SERVER`, `SMTP_PORT`, `SMTP_LOGIN`, `SMTP_PASSWORD`.
```

Comment:
just as an idea - maybe we can also give sample json context of the secret
{ "SMTP_SERVER" : "put_your_host_here",
  "SMTP_PORT" ...
}

Reply:
good idea, I've added this sample, thank you!